llama_stack/log.py

```diff
@@ -137,7 +137,8 @@ def __init__(self, *args, **kwargs):
         # Set a reasonable default width for console output, especially when redirected to files
         console_width = int(os.environ.get("LLAMA_STACK_LOG_WIDTH", "120"))
         # Don't force terminal codes to avoid ANSI escape codes in log files
-        kwargs["console"] = Console(width=console_width)
+        # Ensure logs go to stderr, not stdout
+        kwargs["console"] = Console(width=console_width, stderr=True)
```

Comment:
this is a rather critical fix without which users can't really filter things out correctly. for example right now in trunk `list-deps` for external providers can end up with a lot of noise in the output due to loggers pre-created due to code from external modules not getting silenced. and because the outputs go to stdout, there is no way to filter it via 2>/dev/null !